mne_bids/mne_bids.py

```diff
@@ -623,28 +711,39 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         manufacturer = 'n/a'
 
     # save all meta data
+    # temporarily change write mode for these two files for naming clarity
+    _write_mode = 'append' if write_mode == 'overwrite' else write_mode
+    _participants_tsv(raw, subject_id, "n/a", participants_fname, _write_mode,
+                      verbose)
+    _scans_tsv(raw, os.path.join(kind, raw_file_bids), scans_fname,
+               _write_mode, verbose)
+
+    # condense the clear and overwrite write_mode's
+    if write_mode == 'clear':
+        write_mode = 'overwrite'
+
```

Comment:
perhaps not needed if 'clear' can be 'overwrite' to begin with? Please tell me if I am missing something

Reply:
I felt it pointless to have the private functions that only have the need to overwrite or raise an error to have a 3rd option (ie. `append`), so this line condenses both `overwrite` and `clear` (now `append`) to `overwrite` so that the private functions are clearer.

Reply:
I am resolving this as I think this comment is not relevant anymore.